PilotDataPlatform/metadata
# PILOT-1408 Rename trashed children if parent is renamed

## Summary

- Fix: Children of trashed and auto-renamed folders do not have their paths updated with the new parent name.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1408

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

1. Create a folder and a file within it.
2. Trash the folder.
3. Create the same folder and file again with the same values.
4. Trash the folder again. The folder should be automatically renamed with a timestamp to avoid a naming conflict in the trash bin.
5. Verify that the file has had its path updated with the new folder name.
6. Repeat 1-5 again but check for conflict resolution on restore instead of on trash.
